### ISSUE TITLE
Provide support for Perses dashboards in Mixins.

### ISSUE THREAD

**Issue by nicolastakashi** [prometheus/prometheus#14847]
### Proposal

[Perses](perses.dev) is an open-source project aimed at simplifying the creation and management of dashboards for observability and monitoring. It offers a more flexible and modern alternative to existing solutions like Grafana.

Recently, Perses was accepted into the CNCF Sandbox, marking a significant milestone for the project. Given this momentum, we believe now is the perfect time to start incorporating support for Perses dashboards into the existing monitoring Mixins.

I’d love to hear your thoughts on this approach.

**Comment by gouthamve** [comment]
This is a great idea tbh! I think we should first pilot it in one repository before adopting it widely to see how easy and maintainable the code is. @SuperQ WDYT about the `node_exporter`?

**Comment by SuperQ** [comment]
I have found the jsonnet dashboards 

Similar to how we maintain helm charts in [prometheus-communit/helm-charts](https://github.com/prometheus-community/helm-charts) and Ansible in [prometheus-community/ansible](https://github.com/prometheus-community/ansible). I think we should go with a Perses dashboard monorepo.

**Comment by gouthamve** [comment]
I think it might be better to keep the dashboards, alerts and recording rules alongside the code itself, but we can start with a monorepo to POC this out. It'll even help discoverability if we keep everything in one place initially.

WDYT @nicolastakashi?

**Comment by nicolastakashi** [comment]
I think it's a good idea, maybe something prometheus-community/perses-dashboards, I think is a good start and we can start adding dashboards on that repo.

wdyt @Nexucis ?

**Comment by Nexucis** [comment]
Yeah let's start with a POC. But at the end we should have single repo with the mixins. It will make easier the user life and they will see in a single place what possibility they have. (Also it will highlight better the support of Perses, but no worries I have no interest in Perses ;))

**Comment by Nexucis** [comment]
@nicolastakashi is starting it here: https://github.com/nicolastakashi/community-perses-dashboards.

**Comment by gouthamve** [comment]
Great! Let me know if and when you're ready to move it to `prometheus-community` and I'd be happy to help you migrate it there.

**Comment by beorn7** [comment]
Hello from the bug scrub!

https://github.com/nicolastakashi/community-perses-dashboards looks fairly active. @nicolastakashi how do you feel about moving it to prometheus-community?

**Comment by Nexucis** [comment]
So the repo has moved to the perses org: https://github.com/perses/community-dashboards.

Thanos, Prometheus, Kubernetes and ETCD mixins has been migrated to the Perses format. The question now is more does it makes sense to redistribute these mixins to each owner or to keep it in this repository ?

also FYI, these dashboards are going to be added to the prometheus-operator: https://github.com/prometheus-operator/kube-prometheus/pull/2657

/cc @saswatamcode @heliapb @simonpasquier

**Comment by saswatamcode** [comment]
Love the idea, however, I'm not certain if splitting them up at this stage would make sense.
Also, monitoring mixins as a concept currently only exists with jsonnet, i.e, some jsonnet lib that has all relevant Prometheus rule config + dashboards. So Perses dashboards only being in Go kind of breaks it.

I wonder @Nexucis if we should then,
- Extend community-dashboards repo, to house recording/alerting rule config alongside the Perses dashboard, all in Go
- Import them in kube-prometheus as a way to get people to start using these Go "mixins" (Grafana by default, with flag to switch to Perses)
- Then evaluate splitting across OSS tools, maybe somehow even hooking it into metrics schematization work (can't use a metric in dashboard/rule unless your instrumentation has it! cc: @bwplotka).

**Comment by heliapb** [comment]
Hi, I agree with @saswatamcode - for me at this stage the project will benefit with more with continued development. Alerts seem the next logical step, as well probably continuing support for more dashboards for prometheus exporters and other tools. 

I do feel moving closer to each tool might make sense in the future, as it will probably help with adoption.

**Comment by beorn7** [comment]
Do I understand the sentiment correctly that we shouldn't right now attempt to add Perses support to the existing mixins in the various prometheus/... repos?

Assuming so, I'm closing this, but please let me know if I misunderstood, and I'll promptly re-open.

**Comment by Nexucis** [comment]
Yeah I think you got the correct feeling @beorn7.

perhaps we should keep the repo in perses and then sync the json in other owner repo. I really don't know what is the best for user here.

Regarding the possibility to rewrite alert/recording rule in Golang, what would be the benefit of doing that ? I mean there was a point to rewrite the dashboard because the Perses format is not the same, but for the alert, the format doesn't change. So excepting the fact it would be rewritten in Go, I don't really see the point of doing it.
